tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -15,6 +15,16 @@ Verify Default Access Groups Settings And JupyterLab Notebook Access
     Verify Default Access Groups Settings
     Verify User Can Spawn A Notebook
 
+Verify Empty Group Doesnt Allow Users To Spawn Notebooks
+    [Documentation]   Verify that USer is unable to Access Jupyterhub after modifying Access Groups in rhods-groups-config
+    [Tags]    Sanity
+    ...       ODS-572
```

Comment:
Missing `Tier1` as tag and typo in the documentation (`USer`)

Reply:
Updated